packages/muelu/src/MueCentral/MueLu_SingleLevelFactoryBase.hpp

```diff
@@ -137,7 +137,10 @@ namespace MueLu {
         comm->barrier();
       }
 
-      GetOStream(Test) << *RemoveFactoriesFromList(GetParameterList()) << std::endl;;
+      if (IsPrint(Test1))
+        GetOStream(Test1) << *RemoveFactoriesFromList(GetParameterList()) << std::endl;
+      else
+        RemoveFactoriesFromList(GetParameterList())->print(GetOStream(Test0), Teuchos::ParameterList::PrintOptions().indent(0).showFlags(true).showDefault(false));
```

Comment:
Why is `Test1` necessary?

Reply:
`Test` has been used in a couple of other spots, so I didn't want to change its meaning. `Test1` is now equivalent to printing default values of factory parameters.